codingStandards/AdditionalGuidelines-Java.md

```diff
@@ -0,0 +1,44 @@
+# Additional Guidelines - Java
+
+## Using assertions
```

Comment:
Based on [this](https://docs.docker.com/opensource/doc-style/#capitalization) which we are going to adopt soon, headings should be sentence case. Keep this in sentence case and revise coding standards docs later to match the guideline?